client/components/tinymce/plugins/simple-payments/style.scss

```diff
@@ -75,7 +75,23 @@
 }
 
 .editor-simple-payments-modal__list {
-	padding: 8px 16px;
 	flex: auto;
 	overflow-y: auto;
 }
+
+.editor-simple-payments-modal__list-item {
+	display: flex;
+	align-items: center;
+}
+
+input[type="radio"].editor-simple-payments-modal__list-radio {
```

Comment:
I need to include the `input[type="radio"]` bit here to make the selector more specific. Just the class name wouldn't be enough to override the `input[type="radio"]` rules from `forms.scss`.